app/views/admin/loans/content/_form.html.slim

```diff
@@ -39,10 +45,19 @@
   = render 'admin/loans/content/translatable', loan: @loan, f: f
 
   = authorized_form_field(simple_form: f, model: @loan, field_name: :primary_agent,
-    choices: @agent_choices)
+    choices: @agent_choices, classes: 'hidden-print')
 
   = authorized_form_field(simple_form: f, model: @loan, field_name: :secondary_agent,
-    choices: @agent_choices)
+    choices: @agent_choices, classes: 'hidden-print')
+
+  = f.input 'Primary Agent', wrapper_html: {class: 'print-only'}
+    span = @loan.primary_agent.name
+
+  = f.input 'Secondary Agent', wrapper_html: {class: 'print-only'}
+      span = @loan.secondary_agent.name
+
```

Comment:
Will both primary agent and representative be needed?

Some loans do not have people attached to them, so make sure this code does not fail if an agent or representative is not listed.